qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -175,6 +181,10 @@ def tearDown(self):
         for i, m in enumerate(self.mounts):
             m.client_id = self._original_client_ids[i]
 
+        if self.vol_created:
+            self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", "volume", "rm",
```

Comment:
I don't really want to mix the raw fs commands with the volumes plugin. Let's keep using the raw `ceph fs` commands outside of the `test_volumes.py` code.

Reply:
Done.